.github/workflows/continuous-delivery.yml

```diff
@@ -49,22 +61,22 @@ jobs:
       id: version
       run: |
         echo Releasing as ${{ steps.semanticversion.outputs.new_version }}
-        mvn versions:set -DnewVersion=${{ steps.semanticversion.outputs.new_version }}
+        mvn $MAVEN_CLI_OPTS versions:set -DnewVersion=${{ steps.semanticversion.outputs.new_version }}
 
     - name: Perform build
-      run: mvn -B package --file pom.xml
+      run: mvn $MAVEN_CLI_OPTS package
 
     ## Deploy
     - name: Deploy package
       env:
-            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-            REPOSITORY_URL: ${{ secrets.NEXUS2_REPO_RELEASES  }}
-            REPOSITORY_USERID: ${{ secrets.PUB_NEXUS2_USERNAME_RW }}
-            REPOSITORY_CREDENTIALS: ${{ secrets.PUB_NEXUS2_PASSWORD_RW }}
+            GPG_EXECUTABLE: gpg
+            GPG_SECRET_KEYS: ${{ secrets.LEVIGO_GPG_KEYS }}
+            GPG_OWNERTRUST: ${{ secrets.LEVIGO_GPG_OWNERTRUST }}
+            GPG_PASSPHRASE: ${{ secrets.LEVIGO_GPG_PASSPHRASE }}
       run: |
-        mkdir -p ~/.m2
-        echo "<settings><servers><server><id>neverpile</id><username>${REPOSITORY_USERID}</username><password>${REPOSITORY_CREDENTIALS}</password></server></servers></settings>" > ~/.m2/settings.xml 
-        mvn deploy -Dmaven.test.skip.exec=true -DaltDeploymentRepository=neverpile::default::${REPOSITORY_URL}
+        echo "$GPG_SECRET_KEYS" | base64 --decode | $GPG_EXECUTABLE --import --no-tty --batch --yes
```

Comment:
für was ist denn der Befehl nu wieder da? ;)

Reply:
Kann ja nur 2 Sachen sein: einmal eine Variable damit zuweisen, das sehe ich aber nicht - oder doch den Inhalt der Variable ins Log schreiben, was aber hoffentlich nicht die Intention ist bzw. meines Wissens nach von github doch eh unterdrückt wird?

Reply:
Habe den step einfach vom [ filetype-analyzer](https://github.com/levigo/filetype-analyzer/blob/0c204684dab5239fd5d1e43c4fa49e0f7948581d/.github/workflows/continuous-delivery.yml#L90) übernommen.
Ich denke Mal da werden die Keys und der "Ownertrust" - was auch immer das ist - in das auf der VM installierte GPG importiert, damit sie dann im maven-gpg-plugin verfügbar sind. Und anscheinend sind die secrets base64 encoded.